### ISSUE TITLE
thin white horizontal line in scripting area

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap#2360]
<img width="897" alt="Firefox001" src="https://user-images.githubusercontent.com/3885074/54849335-2bcfe000-4ca1-11e9-9dcb-72418fc4cb74.png">

As I scroll up and down this line appears and disappears.  It's always at the top of the FOR EACH block.  Every third click of my scroll wheel makes it appear; it's not there for the other 2/3 of them.  I can scroll that block all the way offscreen and then back on, and it still appears at that same block position.

**Comment by jmoenig** [comment]
can't reproduce this myself....

**Comment by brianharvey** [comment]
Yeah, it only happened to me this once, but Mary says she's seen it a few times.

**Comment by jmoenig** [comment]
looks like some kind of pixel rounding / truncating error...

**Comment by cycomachead** [comment]
I've seen it off and on for years. A rounding error makes sense to me, but no idea where.
